lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -178,7 +178,7 @@ def send_request(packet, timeout = self.response_timeout)
     if timeout.nil?
       return nil
     elsif response.nil?
-      raise TimeoutError.new("Send timed out")
+      raise Timeout::Error.new("Send timed out")
```

Comment:
I believe this is the intended exception, but I can change this to `Rex::TimeoutError`

The line was last modified here:
https://github.com/rapid7/metasploit-framework/commit/c0494dcbc0657006ff939656b6dbd325e7a69698